File: server/cmd/server/structure/channel/channel.go
package channel

type ChannelData struct {
	ChannelId string `bson:"channelId" json:"id"`
	Name string `json:"name"`
	Org string `json:"org"`
	Type string `json:"type"`
	Photo string `json:"photo"`
	EnglishName string `bson:"englishName" json:"english_name"`
}

File: mobile/lib/models/channel.dart
import 'package:json_annotation/json_annotation.dart';
part 'channel.g.dart';

@JsonSerializable()
class Channel{
  final String channelId;
  final String name;
  final String org;
  final String type;
  final String photo;
  final String englishName;

  Channel(this.channelId, this.name, this.org, this.type, this.photo, this.englishName);
  factory Channel.fromJson(Map<String, dynamic> json) => _$ChannelFromJson(json);
  Map<String, dynamic> toJson() => _$ChannelToJson(this);
}